Tasks/Gradle/CodeAnalysis/SonarQube/endpoint.ts

```diff
@@ -1,5 +1,64 @@
-// Simple data class for a SonarQube generic endpoint
+/// <reference path="../../../../definitions/vsts-task-lib.d.ts" />
+
+import tl = require('vsts-task-lib/task');
+
 export class SonarQubeEndpoint {
     constructor(public Url: string, public Username: string, public Password: string) {
```

Comment:
Reverted: Typescript does not allow the 'private' access modifier on constructors.